ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -22,9 +22,40 @@ parameters:
 - name: BASE_DOMAIN
 - name: CLUSTER_NETWORK_MANIFEST
 - name: ENABLE_FIPS
+- name: IMAGE_EGRESS_HTTP_PROXY
+  required: true
+  value: registry.svc.ci.openshift.org/origin/4.2:egress-http-proxy
+- name: PROXY_ENABLED
```

Comment:
I think we should shift this to `ENABLE_PROXY` to mirror the existing `ENABLE_FIPS`.